server/pbenchinacan/pod.yml

```diff
@@ -1,11 +1,43 @@
-# Created with podman-3.2.2
+#
```

Comment:
This looks very similar to `pod-nodata.yml`.  Filed issue #3041 to track a future effort to DRY it out.